media-autobuild_suite.bat

```diff
@@ -1552,7 +1551,7 @@ start /I %instdir%\%msys2%\usr\bin\mintty.exe -i /msys2.ico -t "media-autobuild_
 --rav1e=%rav1e% --ripgrep=%ripgrep% --dav1d=%dav1d%'
 
 endlocal
-if [%forceQuitBatch%]==[y] taskkill /pid %ourPID% /f
+if [%forceQuitBatch%]==[y] taskkill /f /pid %ourPID% /f
```

Comment:
There's already `/f` there, why are you repeating it?

Reply:
Oops, sorry, didn't see it, fixed now!